src/bin/pg_autoctl/ini_file.c

```diff
@@ -552,8 +552,6 @@ ini_set_setting(const char *filename, IniOption *optionList,
 {
 	IniOption *option = NULL;
 
-	log_debug("Reading configuration from %s", filename);
-
```

Comment:
Why this PR is removing this debug entry?

Reply:
looked like added by me in this PR. restored